lib/plugins/canjs.js

```diff
@@ -76,7 +75,8 @@ can.view.callbacks._tags["can-import"] = function(el, tagData){
 		if(isAPage) {
 			parentMap[name] = false;
 		}
-		root.attr("__renderingAssets").push(name);
+
+		canWait.data({ page: name });
```

Comment:
I think this should be inside the `if(isAPage)` condition, we only need to know about the can-imports that are dynamically importing stuff.